app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_people = [(Person(person.get("name"), person.get("age")))
+                  for person in people]
+    for i, person in enumerate(people):
```

Comment:
You not need enumerate
Check how objects are works. They are mutable or immutable? Do they work by value or link?